### ISSUE TITLE
Is there a problem in the loss function

### ISSUE THREAD

**Issue by hhguo** [mozilla/LPCNet#97]
In your code, the output of your model is logits, you may need to set "from_logits=True" in the loss function "sparse_categorical_crossentropy", or the code will clip the value and process it with log(*).
![image](https://user-images.githubusercontent.com/28179486/71815922-f2475480-30bb-11ea-9b61-4a0798158a59.png)
